lib/dfe/analytics/tasks/import_entities.rake

```diff
@@ -11,6 +13,8 @@ namespace :dfe do
 
     desc 'Send Analytics events for the state of all records in a specified model'
     task :import_entity, %i[entity_name batch_size] => :environment do |_, args|
+      return unless DfE::Analytics.enabled?
```

Comment:
Nitpick - Might be worth having a `puts` here writing to the console, with a message like:
```
DfE Analytics is not enabled - Ignoring import_entity
```

Reply:
Added this 👍